brdnet/run_plier.R

```diff
@@ -33,6 +33,9 @@ parser <- add_argument(parser, 'plierDisease',
 parser <- add_argument(parser, 'outdir', help='The directory to print results to', type='character')
 parser <- add_argument(parser, '-k', help='The number of PLIER PCs to use', 
 					   type='numeric')
+parser <- add_argument(parser, '--noNorm',
+					   help='If this flag is passed, do not normalize the gene expression data',
```

Comment:
Somehow some tabs crept into my code and the interpreter didn't complain because R. It's a github rendering issue, but it also means that something is wrong in my .vimrc somewhere.